lang/proc_macros/src/lib.rs

```diff
@@ -1,12 +1,15 @@
+#![crate_name = "odra_proc_macros"]
```

Comment:
Why?

Reply:
A leftover of some experiments. I'll remove it.